### PR TITLE
chapter_7

### PR DESCRIPTION
**Pull request by maihuysl1999**
Sign up
Kiến thức cần nắm : https://docs.google.com/document/d/1L9dlWmhGjGeCMPiZD5of0TF-LqGJJK9PEh6HX9fVXcY/edit
Screenshot from 2021-08-01 16-00-48

### CODE REVIEW

**Comment by maihuysl1999** [comment]
ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
+  rescue ActiveRecord::RecordNotFound
```
Comment (ngocvt-0484) [inline app/controllers/users_controller.rb]: find by có bắn ra exception k? mà bắt e?

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    flash[:danger] = t ".user_not_found"
+    redirect_to root_url
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
```
Comment (ngocvt-0484) [inline app/controllers/users_controller.rb]:
```suggestion
    @user = User.new user_params
```

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
-    <%= link_to "sample app", "#", id: "logo" %>
+    <%= link_to "sample app", "/", id: "logo" %>
+
     <nav>
       <ul class="nav navbar-nav navbar-right">
-        <li><%= link_to t(".home"), "#" %></li>
-        <li><%= link_to t(".help"), "#" %></li>
-        <li><%= link_to t(".login"), "#" %></li>
+        <li><%= link_to t(".home"), "/" %></li>
+        <li><%= link_to t(".help"), "/static_pages/help" %></li>
```
Comment (ngocvt-0484) [inline app/views/layouts/_header.html.erb]: dùng path
k dùng string

**Comment by maihuysl1999** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**